docs/devSetupOsx.md

```diff
@@ -0,0 +1,51 @@
+# macOS
+
+Auro's development environment uses Node.js tools. If you are contributing to Auro on macOS, [Xcode](https://www.freecodecamp.org/news/how-to-download-and-install-xcode/), [Homebrew](https://brew.sh/) and [Node Version Manager](https://github.com/nvm-sh/nvm#installing-and-updating) highly recommended. Otherwise, you will encounter issues with the repo's tooling.
```

Comment:
```suggestion
Auro's development environment uses Node.js tools. If you are contributing to Auro on macOS, [Xcode](https://www.freecodecamp.org/news/how-to-download-and-install-xcode/), [Homebrew](https://brew.sh/) and [Node Version Manager](https://github.com/nvm-sh/nvm#installing-and-updating) are highly recommended. Otherwise, you will encounter issues with the repo's tooling.
```